src/go/k8s/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -104,6 +104,8 @@ type ClusterSpec struct {
 	// Replicas determine how big the cluster will be.
 	// +kubebuilder:validation:Minimum=0
 	Replicas *int32 `json:"replicas,omitempty"`
+	// PriorityClassName is the name of PriortyClass.
```

Comment:
nit: typo `PriorityClass`

Reply:
nit: can this comment be expanded ever so slightly? Something like `PriorityClassName is used to set the PodSpec.PriorityClassName of the redpanda Statefulset`?

Reply:
Done.